xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/service/BitfinexAccountServiceRaw.java

```diff
@@ -169,14 +198,22 @@ public BitfinexDepositAddressResponse requestDepositAddressRaw(String currency)
     } else if (currency.equalsIgnoreCase("ZEC")) {
       type = "zcash";
     }
-
+    final String finalType = type;
     BitfinexDepositAddressResponse requestDepositAddressResponse =
-        bitfinex.requestDeposit(
-            apiKey,
-            payloadCreator,
-            signatureCreator,
-            new BitfinexDepositAddressRequest(
-                String.valueOf(exchange.getNonceFactory().createValue()), type, "exchange", 0));
+        decorateApiCall(
+                () ->
+                    bitfinex.requestDeposit(
+                        apiKey,
+                        payloadCreator,
+                        signatureCreator,
+                        new BitfinexDepositAddressRequest(
+                            String.valueOf(exchange.getNonceFactory().createValue()),
+                            finalType,
+                            "exchange",
+                            0)))
+            .withRetry(retry("account-depositAddress"))
+            .withRateLimiter(rateLimiter(BITFINEX_RATE_LIMITER))
+            .call();
     if (requestDepositAddressResponse != null) {
```

Comment:
this logic doesn't do anything just return inline, if == null it will just return null.